src/modules/organization/queries.ts

```diff
@@ -8,3 +8,53 @@ export const ALL_INTERNAL_GROUP_POSITIONS = gql`
     }
   }
 `
+export const ALL_INTERNAL_GROUPS = gql`
+  query AllInternalGroups {
+    allInternalGroups {
+      id
+      name
+    }
+  }
+`
+
+export const ALL_INTEREST_GROUPS = gql`
+  query {
+    allInterestGroups {
+      id
+      name
+      type
+      groupImage
+      groupIcon
+    }
+  }
+`
+
+export const ALL_INTERNAL_WORK_GROUPS = gql`
+  query {
+    allInternalWorkGroups {
+      name
+      id
+      groupImage
+      groupIcon
+    }
+  }
+`
```

Comment:
What's this?

Reply:
Internal groups which are type=internal_group and not type=interest_group. Can be removed and replaced by the regular ALL_INTERNAL_GROUPS if we don't need to query both in a single query sometime in the future. I'll replace it for now though